app/views/WhatDoYouWantToDoTileView.scala.html

```diff
@@ -16,20 +16,26 @@
 
 @import controllers.auth.AuthenticatedRequest
 @import includes.card
+@import uk.gov.hmrc.mongoFeatureToggles.services.FeatureFlagService
 @import uk.gov.hmrc.tai.config.ApplicationConfig
+@import uk.gov.hmrc.tai.model.admin.{CyPlusOneToggle, IncomeTaxHistoryToggle}
 @import uk.gov.hmrc.tai.util.{HtmlFormatter, TaxYearRangeUtil => Dates}
 @import uk.gov.hmrc.tai.viewModels.WhatDoYouWantToDoViewModel
 
+@import scala.concurrent.Await
+@import scala.concurrent.duration.{Duration, SECONDS}
 
-@this(mainTemplate: MainTemplate)
+@this(mainTemplate: MainTemplate,
+        featureFlagService: FeatureFlagService)
 
 @(wdywtdForm: Form[uk.gov.hmrc.tai.forms.WhatDoYouWantToDoFormData],
         model: WhatDoYouWantToDoViewModel,
-        appConfig: ApplicationConfig
-)(implicit request: AuthenticatedRequest[_],
+        appConfig: ApplicationConfig)(implicit request: AuthenticatedRequest[_],
         messages: Messages,
-        user: controllers.auth.AuthedUser
-)
+        user: controllers.auth.AuthedUser)
+
+@cyPlusOneToggle = @{Await.result(featureFlagService.get(CyPlusOneToggle), Duration(appConfig.SCAWrapperFutureTimeout, SECONDS))}
+@incomeTaxHistoryToggle = @{Await.result(featureFlagService.get(IncomeTaxHistoryToggle), Duration(appConfig.SCAWrapperFutureTimeout, SECONDS))}
```

Comment:
Don't do this. It is ok ish for sca wrapper because we are going to refactor it later.

Pass the value of the toggle to the view

Reply:
Done, Yeah I tried to keep all SCA thing similar, hence did it like this. but I agree that it should be passed to the view as like other flags.